index.js

```diff
@@ -194,6 +195,10 @@ function getDefaultConfig(mainTemplate, templatesPath) {
         return Array.isArray(value);
       };
 
+      ramlObj.stringify = function (value) {
+        return typeof value === 'object' ? yaml.dump(value) : value;
```

Comment:
@jhen0409 I picked your commit into my PR, and extends it to check the media type.
please see #239. :)